generic/proof-shell.el

```diff
@@ -1610,6 +1610,33 @@ by the filter is to send the next command from the queue."
     ;; sets proof-shell-last-output-kind
     (proof-shell-handle-immediate-output cmd start end flags)
 
+    ;; EXPERIMENTS:
+    ;; There was an error, and probably no goal printed before (becaus
+    ;; eof set silent. So we insert a new Show command, and process it
+    ;; immediately by waiting a few milisec and calling
+    ;; proof-shell-handle-delayed-output.
+    ;; 
+    (when (eq proof-shell-last-output-kind 'error)
+      (proof-add-to-queue
+       (list ;(proof-shell-stop-silent-item)
+        (proof-shell-action-list-item "Show."  'proof-shell-clear-silent)))
+      (setq proof-shell-delayed-output-start (point-max))
+      (setq proof-shell-delayed-output-flags flags)
+      (when t; (proof-shell-exec-loop)
+        (sleep-for 0 100)
```

Comment:
without this it won't work. This suggests it should be put in some callback instead...